sql/mysql/sqlspec.go

```diff
@@ -270,7 +270,12 @@ func convertCharset(spec specutil.Attrer, attrs *[]schema.Attr) error {
 		}
 		*attrs = append(*attrs, &schema.Charset{V: s})
 	}
-	if attr, ok := spec.Attr("collation"); ok {
+	// For backwards compatibility, accepts both "collate" and "collation".
+	attr, ok := spec.Attr("collate")
+	if !ok {
+		attr, ok = spec.Attr("collation")
```

Comment:
Meh. Clutters code. We are pre-release (at least not v1) and can work with API changes, can't we?

Reply:
We actually don't know the impact this will have on users. I think it's fine to mention it in the release notes and for the next minor version (bump to 0.4) we can announce its drop. WDYT?

Reply:
I agree to that. We should given users time to adjust. But we cannot slug every change with us for ever :-)

Reply:
Agree, it's temporary and will be removed in a few weeks from now.